README.md

```diff
@@ -27,52 +27,20 @@ You can check the last version of Back-office here: [kuzzle-backoffice](http://k
 
 For installing the Back-office on your own host, follow next steps.
 
-## With Docker
-### I want to use the all-in-one recipe
-The easiest way to deploy the Kuzzle stack is by using the `docker-compose.yml` file at the root directory of this project. But you can also create your own `docker-compose` in order to change ports or something else.
-
-```
-$ wget https://raw.githubusercontent.com/kuzzleio/kuzzle-build/master/docker-compose/backoffice-docker-compose.yml
-$ docker pull kuzzleio/backoffice
-$ BACKOFFICE_PORT=3000 docker-compose -f backoffice-docker-compose.yml up
+## Use the BO on your server
+You can build the Back-Office with:
 ```
-
->Where `BACKOFFICE_PORT` _(default: 3000)_ corresponding to the port for accessing to the Back-office.
-
-After a while, you can access to `http://<your-ip>:3000`.
-
-### I just want to run the Back-office
-If you already have a Kuzzle stack running somewhere, be sure your Kuzzle Proxy IP and Websocket port (default 7511) is open and accessible from everywhere.
-
-```
-$ docker pull kuzzleio/backoffice
-$ docker run -p 3000:3000 -e "BACKOFFICE_PORT=3000" kuzzleio/backoffice
-```
->`BACKOFFICE_PORT` _(default: 3000)_ corresponding to the port for accessing to the Back-office.
-
-## Without Docker
-Be sure to have node v4.4 and bower installed.
-
-```
-$ git clone https://github.com/kuzzleio/kuzzle-backoffice
-$ npm install
-$ bower install
-```
-
-At this point you can choose whether to run with the embed server or without:
-
-### With the embed server
-```
-$ npm run build && BACKOFFICE_PORT=3000 npm start
+$ npm run install_deps
+$ npm run build
 ```
->`BACKOFFICE_PORT` _(default: 3000)_ corresponding to the port for accessing to the Back-office.
 
-You can now access `http://<back-office-ip>:<BACKOFFICE_PORT>`
+The `dist` folder is now generated, you can expose it through a web server (like nginx) in order to expose it.
```

Comment:
Quick note: we can't access to the BO without a dev server.
BTW, I fixed with your proposal